model/perf_data.go

```diff
@@ -103,7 +101,7 @@ func (r *PerfRollups) Setup(env cedar.Environment) {
 	r.env = env
 }
 
-func (r *PerfRollups) Add(ctx context.Context, name string, version int, userSubmitted bool, t MetricType, value interface{}) error {
+func (r *PerfRollups) Add(ctx context.Context, name string, version int, userSubmitted, valid bool, t MetricType, value interface{}) error {
```

Comment:
I feel like two bools in a row is hard to read, the arguments to this function feel like they're getting a little out of control. Should we try and use an arg-struct? or pass a PerfRollupValue directly?